packages/rich-text/src/to-dom.js

```diff
@@ -110,6 +105,33 @@ function remove( node ) {
 	return node.parentNode.removeChild( node );
 }
 
+function createBogusBR( doc ) {
```

Comment:
If 'br' is an abbreviation of "break" (unsure), correct capitalization would be `createBogusBr`